site.yml

```diff
@@ -1,4 +1,48 @@
 ---
-- include: redhat_release.yml
-- include: autotested.yml
-- include: cleanup.yml
+
+# Setup/override global variables
+- hosts: all
+  tasks:
+    # Defines adept_job_no_group and rt_adept_job_no
+    - include: roles/common/tasks/adept_job_no.yml
+
```

Comment:
What would you think of:

    vars:
      some_descriptive_variable_name: "{{ groups[adept_job_no_group] }}"